.github/workflows/build_native_runtime.yml

```diff
@@ -0,0 +1,42 @@
+name: Build Native Runtime
+
+on:
+  push:
+    branches: [ master ]
+
+  pull_request:
+    branches: [ master ]
+
+
+jobs:
+  build_native_runtime:
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [macos-latest, ubuntu-18.04]
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+          submodules: recursive
+
+      - name: Set up JDK 11.0.8
+        uses: actions/setup-java@v1
+        with:
+          java-version: 11.0.8
+
+      - name: Run CMake
+        run: |
+          mkdir build
+          cd build
+          cmake -B . -S ../nativeruntime/cpp
+          make
+
+      - name: Upload libnativeruntime
+        if: always()
+        uses: actions/upload-artifact@v2
+        with:
+          name: libnativeruntime-${{ matrix.os }}
```

Comment:
The `matrix.os` has OS version, and the generated artifacts have this version suffix, for example ` libnativeruntime-macos-latest ` and `libnativeruntime-ubuntu-18.04`. It's better if we can change suffix to macOS/Linux, and add architecture directory, such as x86_64, arm, etc.

Reply:
Latest commit did it.